src/v2/Apps/Conversation/Routes/Conversation/index.tsx

```diff
@@ -27,6 +27,9 @@ interface ConversationRouteProps {
 const ConstrainedHeightFlex = styled(Flex)`
   height: calc(100vh - 145px);
   ${media.md`
+   height: calc(100vh - 85px);
+ `}
```

Comment:
Gives us a little breathing room for the keyboard to expand on medium sized screens (iPad pro), this issue was happening on tablet Chrome.

Before fix:
<img width="500" alt="Screen Shot 2020-07-22 at 5 15 12 PM" src="https://user-images.githubusercontent.com/12748344/88229560-ee85b480-cc3e-11ea-808f-cb417168ee2a.png">

<br>
<br>

After fix:
<img width="500" alt="Screen Shot 2020-07-22 at 5 19 41 PM" src="https://user-images.githubusercontent.com/12748344/88229924-8d121580-cc3f-11ea-82f4-07e3a7c441c1.png">